03_letters/nicholascross/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func sortLetters(m map[rune]int) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
```

Comment:
Improvement: Use range with key value pair. Then no need to access map again using `m[k]`
```
for k, v := range m {
  ...
}
```

Reply:
Thanks - that is better.  I will update 👍